### DISCUSSION TITLE
Foundations course: Suggestion to add a solution for an error when cloning the repo

### DISCUSSION THREAD

**Discussion by ronshalit00** [TheOdinProject/curriculum#25452] [General]
### Describe your suggestion

Hi, in the assignment of the [DOM Manipulation And Events](https://www.theodinproject.com/lessons/foundations-dom-manipulation-and-events#assignment), 
when cloning the forked repo , I had this error:

> Cloning into 'JavaScript30'...
> remote: Enumerating objects: 1441, done.
> client_loop: send disconnect: Broken pipe
> fetch-pack: unexpected disconnect while reading sideband packet
> fatal: early EOF
> fatal: fetch-pack: invalid index-pack output

* I am using XUbuntu on virtualbox.

After searching the web I found a solution that worked for me: `git config --global core.compression 0
` and then trying to clone.

My suggestion is to add that solution for the next learners that might run into that error, thanks!

### Path

Foundations

### Lesson Url

https://www.theodinproject.com/lessons/foundations-dom-manipulation-and-events#assignment

### Checks

- [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
- [ ] Would you like to work on this issue?

### (Optional) Discord Name

RonS#9861

**Comment by KevinMulhern** [comment]
Hey @ronshalit00, thanks for reporting. I've had a look through our support channels but haven't been able to find any occurrences of this happening for our other learners. Thats not to say it hasn't happened to any of our learners, but it's a pretty good indication of how rare this probably this.

We'll leave this open for a while to see if anyone else runs into it and hopefully comments.
